firmware/steering/kia_soul_ps/steering_control_module.ino

```diff
@@ -266,14 +301,17 @@ void calculateTorqueSpoof( float torque, uint16_t * TSpoofL, uint16_t * TSpoofH
 // A function to parse incoming serial bytes
 void processSerialByte( uint8_t incomingSerialByte) 
```

Comment:
We removed this function in the main branch I think. Nbd as Petes PR should remove it, but just noting it.